test/acceptance/features/steps/steps.py

```diff
@@ -278,6 +278,43 @@ def then_envFrom_contains_intermediate_secret_name(context, intermediate_secret_
         intermediate_secret_name, wait=True, timeout=120) is not None, f"There is no deployment with intermediate secret {intermediate_secret_name}"
 
 
+# STEP
+@given(u'OLM Operator Backend is running')
+def given_olm_operator_backend(context):
+    openshift = Openshift()
+    _ = openshift.oc_apply_yaml_file(os.path.join(os.getcwd(), "test/acceptance/resources/backend.operator.manifest.yaml"))
+
+
+# STEP
+@given(u'Backend CR "{cr_name}" is applied')
+def create_backend_cr(context, cr_name):
+    create_cr(context, cr_name)
+
+
+# STEP
+@given(u'Service Binding request is applied')
```

Comment:
@Avni-Sharma and @pmacik How about making available [sbr creation step title](https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/steps.py#L144) more generic to support all the case we'll have. Something like just `Service Binding Request is applied`? So we won't have to write small gratuitous functions to call the available function.

Reply:
yes, we can do this, but requires to change here other unrelated test. IMHO, I would do this consolidation in a separate PR.

Reply:
Cool, I'm fine with it.